tmk_core/common/chibios/gpio.h

```diff
@@ -32,3 +34,17 @@ typedef ioline_t pin_t;
 #define readPin(pin) palReadLine(pin)
 
 #define togglePin(pin) palToggleLine(pin)
+
+/* Operation of GPIO by port. */
+
+typedef uint16_t port_data_t;
+
+#define readPort(qmk_pin) palReadPort(PAL_PORT(qmk_pin))
```

Comment:
Any reason for using `qmk_pin` over `pin`?

Reply:
I wanted to emphasize that `pin` is a QMK term, so in `tmk_core/common/chibios/gpio.h` I used `qmk_pin`.   However, it is not consistent with `pin` in the first half of `tmk_core/common/chibios/gpio.h`.

OK, change `qmk_pin` to `pin`.